examples/helloworld/client.py

```diff
@@ -128,7 +128,7 @@ def _as_dict(data: typing.Any, *, convert_all: bool = True) -> dict[str, typing.
     if not dataclasses.is_dataclass(data):
         raise TypeError(f"{data} must be a dataclass")
     if convert_all:
-        result = typing.cast(dict[str, typing.Any], dataclasses.asdict(data))
+        result = dataclasses.asdict(data) # type: ignore[call-overload]
```

Comment:
nit: add an extraspace before `# type: ignore[call-overload]` to comply with PEP 8 E261 (at least two spaces before inline comment)